CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 - Added `Icmp` and `IcmpV6` to `SockProtocol`.
   (#[2103](https://github.com/nix-rust/nix/pull/2103))
+- `sys::mman::mmap` to return `NonNull<c_void>`.
+  ([#2000](https://github.com/nix-rust/nix/pull/2000))
```

Comment:
This change should be put under `### Changed` rather than `### Added`?

Reply:
Changed.